src/modules/firebase-auth/firebase-validate.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common';
+import * as firebase from 'firebase-admin';
+
+const firebase_params = {
+  type: process.env.TYPE,
+  projectId: process.env.PROJECT_ID,
+  privateKeyId: process.env.PRIVATE_KEY_ID,
+  privateKey: process.env.PRIVATE_KEY.replace(/\\n/g, '\n'),
```

Comment:
Why do we have "\n"s in our `PRIVATE_KEY`? We shouldn't process the config options.